docs/_includes/generated/netty_shuffle_environment_configuration.html

```diff
@@ -20,6 +20,12 @@
             <td>Boolean</td>
             <td>Enable SSL support for the taskmanager data transport. This is applicable only when the global flag for internal SSL (security.ssl.internal.enabled) is set to true</td>
         </tr>
+        <tr>
```

Comment:
We introduce two options below, why the generated html has only one new option?

Reply:
Another option is annotated as ExcludeFromDocumentation because the  LZ4 is the only option. This option won't be documented until we make the compression pluggable.